R/RunDiagnostics.R

```diff
@@ -232,7 +236,9 @@ executeDiagnostics <- function(cohortDefinitionSet,
                                sampleN = 1000,
                                seed = 64374,
                                seedArgs = NULL,
-                               sampleIdentifierExpression = "cohortId * 1000 + seed") {
+                               sampleIdentifierExpression = "cohortId * 1000 + seed",
+                               useAchilles = FALSE, 
+                               achillesDatabaseSchema) {
```

Comment:
should this be achillesDatabaseSchema = resultsDatabaseSchema?

Reply:
resultsDatabaseSchema is not a parameter to executeDiagnostics though. We could set the default to `achillesDatabaseSchema = cdmDatabaseSchema`